decidim-conferences/app/serializers/decidim/conferences/download_your_data_conference_registration_serializer.rb

```diff
@@ -7,15 +7,15 @@ class DownloadYourDataConferenceRegistrationSerializer < Decidim::Exporters::Ser
       def serialize
         {
           id: resource.id,
-          user: {
```

Comment:
I guess this doesn't make sense to be exported is the person that exports this is actual user (ex: current_user).

Reply:
Same as the other comment :) 

My idea is to remove it from all the DownloadYourData, because it doesn't make sense to have this information here